### PR TITLE
[5.0] Remove tfa language files

### PR DESCRIPTION
**Pull request by brianteeman**
Removes the tfa language files as they have been replaced with mfa language files

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on c354b4303790cbb4b5ab3fc63b81f4c64c6d3315<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41574">issues.joomla.org/tracker/joomla-cms/41574</a>.</sub>

File: administrator/components/com_admin/script.php [hunk 1]
```diff
@@ -2213,6 +2213,11 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
             '/templates/system/incompatible.html,',
             '/media/plg_system_compat',
             '/media/plg_editors_tinymce/js/plugins/highlighter',
+            // From 5.0.0-alpha4 to 5.0.0-alpha5
+            'administrator/language/en-GB/plg_twofactorauth_totp.ini',
+            'administrator/language/en-GB/plg_twofactorauth_totp.sys.ini',
+            'administrator/language/en-GB/plg_twofactorauth_yubikey.ini',
+            'administrator/language/en-GB/plg_twofactorauth_yubikey.sys.ini',
```
Comment (heelc29) [inline administrator/components/com_admin/script.php]:
```suggestion
            // From 5.0.0-alpha4 to 5.0.0-beta1
            '/administrator/language/en-GB/plg_twofactorauth_totp.ini',
            '/administrator/language/en-GB/plg_twofactorauth_totp.sys.ini',
            '/administrator/language/en-GB/plg_twofactorauth_yubikey.ini',
            '/administrator/language/en-GB/plg_twofactorauth_yubikey.sys.ini',
```
next version is beta1?

File: administrator/components/com_admin/script.php [hunk 2]
```diff
@@ -2213,6 +2213,11 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
             '/templates/system/incompatible.html,',
             '/media/plg_system_compat',
             '/media/plg_editors_tinymce/js/plugins/highlighter',
+            // From 5.0.0-alpha4 to 5.0.0-beta1
+            '/administrator/language/en-GB/plg_twofactorauth_totp.ini',
+            '/administrator/language/en-GB/plg_twofactorauth_totp.sys.ini',
+            '/administrator/language/en-GB/plg_twofactorauth_yubikey.ini',
+            '/administrator/language/en-GB/plg_twofactorauth_yubikey.sys.ini',
```
Comment (richard67) [inline administrator/components/com_admin/script.php]: @brianteeman You have added the files to the `$folders` and not to the `$files` array, this is wrong.

Comment (richard67) [reply administrator/components/com_admin/script.php]: You have to add them here: https://github.com/joomla/joomla-cms/blob/65a1de2adeb8211b8ace61fb1d86af522b6d63e1/administrator/components/com_admin/script.php#L1997

**Comment by richard67** [comment]
I have tested this item :red_circle: unsuccessfully on 65a1de2adeb8211b8ace61fb1d86af522b6d63e1<br /><br />See my review comment.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41574">issues.joomla.org/tracker/joomla-cms/41574</a>.</sub>

**Comment by brianteeman** [comment]
DOH . Will fix in the morning

**Comment by brianteeman** [comment]
Corrected the silly error

**Comment by brianteeman** [comment]
@richard67 are these two line correct??
```
             // From 5.0.0-alpha3 to 5.0.0-alpha4
            '/templates/system/incompatible.html,/includes',
            '/templates/system/incompatible.html,',
```

**Comment by richard67** [comment]
> @richard67 are these two line correct??
> 
> ```
>              // From 5.0.0-alpha3 to 5.0.0-alpha4
>             '/templates/system/incompatible.html,/includes',
>             '/templates/system/incompatible.html,',
> ```

Yes they are. See https://github.com/joomla/joomla-cms/pull/41393#discussion_r1299177607 and #41321, or check a beta 3 zip package. the folder with the comma at the end of the name was really created that way.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 6d11c98d2a5cb474c90fe436f71d9157c6f9c823<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41574">issues.joomla.org/tracker/joomla-cms/41574</a>.</sub>

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on 6d11c98d2a5cb474c90fe436f71d9157c6f9c823<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41574">issues.joomla.org/tracker/joomla-cms/41574</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41574">issues.joomla.org/tracker/joomla-cms/41574</a>.</sub>

**Pull request merged**